tests/bsim/bluetooth/host/l2cap/multilink_peripheral/src/data.h

```diff
@@ -0,0 +1,14 @@
+/*
+ * Copyright (c) 2024 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef ZEPHYR_TESTS_BSIM_BLUETOOTH_HOST_L2CAP_MULTILINK_PERIPHERAL_SRC_DATA_H_
+#define ZEPHYR_TESTS_BSIM_BLUETOOTH_HOST_L2CAP_MULTILINK_PERIPHERAL_SRC_DATA_H_
+
+#define SDU_NUM  200          /* Number of SDU (L2CAP buffers) to send  */
+#define SDU_LEN  20           /* Length in bytes of said SDUs */
+#define DUT_NAME "dee-yu-tee" /* name to advertise with */
```

Comment:
Why not use the BT name?

Reply:
To avoid the name-in-ad shenanigans in the host. It's clearer IMO to have `BT_DATA(BT_DATA_NAME_COMPLETE, DUT_NAME,)` than to need to have this comment: https://github.com/jori-nordic/zephyr/blob/737e099e3932b5b64a2b12cbaf7238d31b41a2b6/tests/bsim/bluetooth/host/misc/sample_test/src/dut.c#L107-L112

Reply:
Right, but that comment applies to the central, and this is for the peripheral, right? 

But why not define it as
```c
#define DUT_NAME CONFIG_BT_DEVICE_NAME
```
then?